libparsec/crates/types/src/local_manifest.rs

```diff
@@ -100,7 +101,7 @@ impl Chunk {
             // TODO: what to do with overflow
             stop: (block_access.offset + block_access.size.get())
                 .try_into()
-                .unwrap_or_else(|_| unreachable!()),
+                .expect("Chunk `stop` integer should not be zero"),
```

Comment:
On top of the `block_access.size` is `NonZeroU64` so the predicate is already checked

Reply:
@mmmarcos up

Reply:
I updated the message to be a little more precise about the error

Reply:
I didn't see the comment above, maybe you published them only recently  :thinking: